model/task/task.go

```diff
@@ -1444,14 +1445,24 @@ func FindSchedulableForAlias(id string) ([]Task, error) {
 
 func FindRunnable(distroID string, removeDeps bool) ([]Task, error) {
 	match := scheduleableTasksQuery()
+	var d distro.Distro
+	var err error
 	if distroID != "" {
 		match[DistroIdKey] = distroID
+		d, _ = distro.FindOne(distro.ById(distroID).WithFields(distro.ValidProjectsKey))
```

Comment:
You're ignoring the error here but then checking it

Reply:
fixed